meta-ros-common/conf/ros-distro/include/enable-fortran.inc

```diff
@@ -1,9 +1,9 @@
-# Copyright (c) 2019 LG Electronics, Inc.
+# Copyright (c) 2019-2020 LG Electronics, Inc.
 
 # FORTRAN support (copied from conf/local.conf.sample.extended)
 # Enabling FORTRAN
 # Note this is not officially supported and is just illustrated here to
 # show an example of how it can be done
 # You'll also need your fortran recipe to depend on libgfortran
```

Comment:
> You'll also need your fortran recipe to depend on libgfortran

Which fortrain recipe?

Reply:
I think it refers to the recipe you're trying to build with fortran, it was added in 2013:
https://git.openembedded.org/openembedded-core/commit/?id=ae8c17be2845eff2be8394a5d9a45e6aa321c33d